readme.md

```diff
@@ -65,6 +65,14 @@ Returns a [`child_process` instance](https://nodejs.org/api/child_process.html#c
 
 The `child_process` instance is enhanced to also be promise for a result object with `stdout` and `stderr` properties.
 
+### execa.stdout(file, [arguments], [options])
+
+Same as `execa()`, but returns only `stdout`.
+
```

Comment:
No source, I just assumed if you ignore it it dumps the output and does not concat it all together in memory. I think it's a safe assumption.